### PR TITLE
update mco operator display name in CSV.

### PR DESCRIPTION
**Pull request by morvencao**
Signed-off-by: morvencao <lcao@redhat.com>

### CODE REVIEW

File: bundle/manifests/multicluster-observability-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -58,17 +58,17 @@ spec:
   customresourcedefinitions:
     owned:
     - description: MultiClusterObservability defines the configuration for the Observability installation on Hub and Managed Clusters all through this one custom resource.
-      displayName: Multi Cluster Observability
```
Comment (clyang82) [inline bundle/manifests/multicluster-observability-operator.clusterserviceversion.yaml:61]: How does it generate?

Comment (morvencao) [reply bundle/manifests/multicluster-observability-operator.clusterserviceversion.yaml:61]: Looks like it automatically divide the word by camel case, not sure about that.

**Comment by morvencao** [comment]
Updated the changeset to use the operator-sdk marker to customize the display name.

**Comment by morvencao** [comment]
@clyang82 another look?

**Pull request merged**